db/migrate/20180606143538_create_bus_stop_fields.rb

```diff
@@ -0,0 +1,10 @@
+class CreateBusStopFields < ActiveRecord::Migration[5.1]
+  def change
+    create_table :bus_stop_fields do |t|
+      t.string :value
+
+      t.timestamps
+    end
+    create_join_table :bus_stops, :fields
```

Comment:
Does this not attempt to create the table twice? I would have expected just a singular `create_table`, since `create_join_table` doesn't allow specifying extra fields.

Reply:
I found this online: 
[`create_join_table(table_1, table_2, options = {}) public`](https://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/create_join_table)
That's why I put two fields down. Am I misinterpreting what the two tables are?

Reply:
Do you want this migration to create two tables, though? My reading is that the first `create_table` makes a table called `bus_stop_fields`, and the `create_join_table` creates a table called `bus_stops_fields`, and with no `value` column.

Neither of those seems correct to me, but in either event I assume you don't want both.

Reply:
AH. I get what you mean. Thanks. I'll work on it tomorrow

Reply:
Actually, I don't understand why the first table is wrong.

Reply:
Typically join tables would have the pluralized names of both tables. As you've written it, `bus_stop` is singular.